### ISSUE TITLE
How to use transfer learning in tts_train.py

### ISSUE THREAD

**Issue by JoeyHeisenberg** [espnet/espnet#1401]
After training the fastspeech.v3 model with baker dataset, I would like to retrain the model with another but smaller dataset, I only see "resume" option in tts_train.py, it that mean I only can load the model in snapshot but not in model.loss.best or model.last1.avg.best (second question is what is the differents between model.loss.best and model.last1.avg.best or how to generate the model.last1.avg.best? didn't find that code)

**Comment by kan-bayashi** [comment]
`model.last1.avg.best` is generated from here:
https://github.com/espnet/espnet/blob/2b9e8476b357eee78c0d5896833eeaeb0d1d5efe/egs/csmsc/tts1/run.sh#L168
This model parameters are average of the last N epochs. (last1 means just a final epoch.)

And there is `--pretrain-model` option for Tacotron 2 and Transformer.
Not yet added in FastSpeech but it is quite easy.
Please add following lines in `espnet/nets/pytorch_backend/e2e_tts_fastspeech.py`.
https://github.com/espnet/espnet/blob/2b9e8476b357eee78c0d5896833eeaeb0d1d5efe/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py#L302-L303
https://github.com/espnet/espnet/blob/2b9e8476b357eee78c0d5896833eeaeb0d1d5efe/espnet/nets/pytorch_backend/e2e_tts_tacotron2.py#L468-L470
And then you can add `pretrain-model: <your model path>` in your yaml config.

**Comment by JoeyHeisenberg** [comment]
Thx     : ）
@kan-bayashi
